contacts-app/server.js

```diff
@@ -0,0 +1,58 @@
+var express = require('express');
+var app = express();
+
+var mongojs = require('mongojs');
+var db=mongojs('contacts',['contacts']);
+
+var bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+app.get('/', function (req,res) {
```

Comment:
all routes should go to separate files so create a new directory called `routes` and routes files there 

fyi route file is a controller file  and vice versa